der-moderne-staat.csl

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>der moderne staat (German)</title>
+    <title-short>dms</title-short>
+    <id>http://www.zotero.org/styles/der-moderne-staat</id>
+    <link href="http://www.zotero.org/styles/der-moderne-staat" rel="self"/>
+    <link href="http://www.budrich-journals.de/index.php/dms/about/submissions#authorGuidelines" rel="documentation"/>
+    <author>
+      <name>Sebastian Haunss</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="political_science"/>
+    <issnl>1865-7192</issnl>
+    <summary>der moderne staat</summary>
+    <updated>2015-01-29T12:46:57+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="editor" form="short">Hrsg.</term>
```

Comment:
That should already be the standard in the local file and you don't need to define it here seperately.